fe/fe-core/src/main/java/com/starrocks/catalog/BaseTableInfo.java

```diff
@@ -183,6 +183,10 @@ public Table getTableByName() {
             LOG.warn("catalog {} not exist", catalogName);
             return null;
         }
+        // upgrade from 3.1 to 3.2, dbName/tableName maybe null after dbs or tables are dropped
```

Comment:
another question is why this function is supported in this class. Having a function called global single instance in such a basic class is unreasonable. This will cause class dependence too messy and bug-prune.

Reply:
it is a history problem. refactor is another problem. should I refactor it in another pr?

Reply:
Please refactor this later, the current implementation is error-prune.